bears/shell/ShellCheckBear.py

```diff
@@ -20,8 +20,14 @@ class ShellCheckBear:
     CAN_DETECT = {'Syntax', 'Security', 'Undefined Element', 'Unused Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file, shell: str='sh'):
+    def create_arguments(filename, file, config_file, shell: str='sh',
+                         ignore_rules: list=[]):
```

Comment:
`ignore_rules` should be be `shellcheck_ignore` or `shellcheck_ignore_codes`.
See `$ git grep _ignore`